test/llvm/debugInfo/gdb/basicTypes.suppressif

```diff
@@ -0,0 +1,3 @@
+# running gdb on multi-locale code is not yet well supported
+CHPL_COMM!=none
+COMPOPTS <= --no-local
```

Comment:
Why not `skipif` these instead of suppressing?

Reply:
Because ideally, one day they work. These are copied from the existing `targetVar.suppressif`